### ISSUE TITLE
(DB) Query method endAt() seems to be ignored

### ISSUE THREAD

**Issue by GV3** [myflashlab/Firebase-ANE#16]
Hi Guys, 

I'm having a strange BUG with the Firebase DB ANE.

The Queries are completely ignoring anything passed via endAt() or startAt()

Example, let's say I have this structure:
```

 - house:
        room1:
               chair1
                     position: 1
               chair2
                     position: 2
               chair3
                     position: 3
               chair4
                     position: 4
               chair5
                     position: 5
               chair6
                     position: 6
               chair7
                     position: 7
               chair8
                     position: 8
               chair9
                     position: 9
               chair10
                     position: 10
               chair11
                     position: 11
               chair12
                     position: 12
               chair13
                     position: 13
               chair14
                     position: 14
               chair15
                     position: 15
               chair16
                     position: 16
               chair17
                     position: 17
               chair18
                     position: 18
               chair19
                     position: 19
               chair20
                     position: 20

```
This query for example:

`DB.getReference('house').child('room1').orderByChild('position').endAt(10).limitToLast(5);`

SHOULD retrieve **chairs 5 to 10**.

Instead, it retrieves **chairs 15 to 20**.

Af if this would be the query:

`DB.getReference('house').child('room1').orderByChild('position').limitToLast(5);`

basically ignoring the `endAt() `method

Could you please try to reproduce?

I have tried any combination using the endAt and startAt and none seem to work

**Comment by GV3** [comment]
Hope you take into consideration this bug report in the coming urgent DB update :)

**Comment by myflashlab** [comment]
Is this happening on Android or iOS?

**Comment by myflashlab** [comment]
Ok, got it, this is happening on iOS. The Android side works fine though! We're on it now.

**Comment by myflashlab** [comment]
Ok, the problem is fixed now. Thanks for reporting that. the new version will be released in a couple of days with having this issue fixed.

**Comment by GV3** [comment]
Thank you!
Looking forward to the soon release of this new version.
